packages/core/src/models/lessonPlans.ts

```diff
@@ -65,7 +65,7 @@ export class LessonPlans {
   private _prisma: PrismaClientWithAccelerate;
   constructor(private readonly prisma: PrismaClientWithAccelerate) {
     this._prisma = prisma;
-    this._rag = new RAG(this._prisma);
+    this._rag = new RAG(this._prisma, { chatId: "none" });
```

Comment:
Models are mainly used for ingest, so we don't have a chatId or a userId

Reply:
I wondered about having a special string for this. We might want to do same for userId at some point? Perhaps not

Reply:
Yes we might want to also store something for the user ID in non-user calls. I'm expecting we look through the database at some point to see where we are missing what we need, and that "none" should mean we can see those rows. I had to pass a string here because the underlying method now throws if chatId is not supplied.